rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -101,20 +181,473 @@ def test_hop_optimism_eth_receive_no_event(database, optimism_inquirer, optimism
+            counterparty=CPT_HOP,
+            address=MAGIC_AMM_WRAPPER,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xebe61c49901a46c954e37B8945Bfb87D238F8f45']])
+def test_hop_usdc_bridge_l2_to_l1_ethereum(database, ethereum_inquirer: 'EthereumInquirer', ethereum_accounts):  # noqa: E501
+    tx_hash = deserialize_evm_tx_hash('0xc76dbb99825d1e53488426b5cf93f7ff71b5fd1fccd41259537e84824a950cc9')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1715208275000)
+    bridge_amount = '60786.713495'
+    user_address = ethereum_accounts[0]
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=182,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_USDC,
+            balance=Balance(amount=FVal(bridge_amount)),
+            location_label=user_address,
+            notes=f'Bridge {bridge_amount} USDC via Hop protocol',
+            counterparty=CPT_HOP,
+            address=USDC_BRIDGE,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('gnosis_accounts', [['0xebe61c49901a46c954e37B8945Bfb87D238F8f45']])
+def test_hop_usdc_bridge_l2_to_l1_gnosis(database, gnosis_inquirer: 'GnosisInquirer', gnosis_accounts):  # noqa: E501
+    tx_hash = deserialize_evm_tx_hash('0x0c6084399c873b06407f073acec89ffd9693ac0c7df4befd9e45e4178b5ae869')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=gnosis_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1715204760000)
+    bridge_amount = '60786.713495'
+    gas_fees = '0.0001299435'
+    user_address = gnosis_accounts[0]
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_XDAI,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=user_address,
+            notes=f'Burned {gas_fees} XDAI for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=462,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_HUSDC_GNOSIS,
+            balance=Balance(amount=FVal(bridge_amount)),
+            location_label=user_address,
+            notes=f'Burn {bridge_amount} of Hop hUSDC',
```

Comment:
Burn? Isn't this a bridge?

Reply:
okay so for this I will just leave it like this and see how users see it and we can adjust